kerastuner/tuners/hyperband.py

```diff
@@ -11,29 +11,43 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import copy
+import math
 import random
 
 from ..engine import multi_execution_tuner
 from ..engine import oracle as oracle_module
-from ..engine import trial as trial_lib
 
 
 class HyperbandOracle(oracle_module.Oracle):
     """Oracle class for Hyperband.
 
+    Note that to use this Oracle with your own subclassed Tuner, your Tuner
+    class must understand three special hyperparameters that will optionally
+    be set by this Tuner:
+
+      - tuner/trial_id: The trial_id of the Trial to load from when starting
```

Comment:
Use string delimiters around hp names to make it clear that "x/y" is  the full name

Reply:
Done